app/frontend/src/components/designer/FormDesigner.vue

```diff
@@ -431,11 +430,13 @@ export default {
           this.canSave = true;
           modified?.components?.map((comp) => {
             if (comp.key === 'form') {
-              this.addNotification({
-                ...NotificationTypes.ERROR,
-                message: this.$t('trans.formDesigner.fieldnameError', {
-                  label: comp.label,
-                }),
+              const notificationStore = useNotificationStore();
+              const msg = i18n.t('trans.formDesigner.fieldnameError', {
+                label: comp.label,
+              });
+              notificationStore.addNotification({
+                text: msg,
```

Comment:
It looks like "message" is still being used in EmailTemplate and UserSubmission - if these also need to be updated, would it be better to put them in this PR, or to open new Jira item(s)?

Reply:
let me find that and deal with it in this PR. Great find!

Reply:
Looks great!